curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/delete-one-document-using-model.findbyidandremove.english.md

```diff
@@ -6,7 +6,7 @@ challengeType: 2
 
 ## Description
 <section id='description'>
-Delete one person by her _id. You should use one of the methods findByIdAndRemove() or findOneAndRemove(). They are like the previous update methods. They pass the removed document to the cb. As usual, use the function argument personId as search key.
+Delete one person by the person's <code>_id</code>. You should use one of the methods <code>findByIdAndRemove()</code> or <code>findOneAndRemove()</code>. They are like the previous update methods. They pass the removed document to the cb. As usual, use the function argument <code>personId</code> as the search key.
```

Comment:
```suggestion
In this challenge you will need to delete a person using their <code>_id</code>
```
This looks like a better description

Reply:
`their` refers to more than one person, which is why I wrote it this way.